docker-compose.yml

```diff
@@ -90,6 +90,22 @@ services:
     networks:
       - psp
 
+  ####################### Clam AV #######################
+  # example av test file here: https://serverfault.com/questions/484082/how-should-i-test-clam-anti-virus
```

Comment:
My computer keeps removing this test file so I assume others would have the same problem. Link to instructions on how to create that file instead.